v4.13/catalog-template.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Do we need to maintain the files under v4.13? Is there any update like v4.13 -> v4.14 in the future?

Reply:
no. it is just a sample to ensure the fbc pipeline test can be passed.

Reply:
If it is just a sample, maybe it is better to just put it in a folder like sample, examples?
Otherwise I would think it is a main part of the repo.